### ISSUE TITLE
thumbnails and thumbnail links missing alt text

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6749]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.0), thumbnails and thumbnail link do not have alt text and therefore fail to meet WCAG AA accessibility requirements.

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to dev.nurax.samvera.org and run a search
3. Run the Site Improve browser extension on the search results page

![image_text_alt](https://github.com/samvera/hyrax/assets/32644254/6fc22933-9c34-4b7e-9eb8-e4fc2093b3f5)

![link_text_alt](https://github.com/samvera/hyrax/assets/32644254/a18eae6e-5b95-407b-9eb8-00d95db97d0b)
